### PR TITLE
Support debugging networking tests with delve

### PR DESCRIPTION
**Pull request by marun**

### CODE REVIEW

**Comment by marun** [comment]
Whoops, left off the test args in the previous patchset.

**Comment by pravisankar** [comment]
https://github.com/marun/origin/blob/nw-test-debug/test/extended/networking.sh#L26 typo *networking

**Comment by pravisankar** [comment]
When OPENSHIFT_USE_DEBUGGER=true, it will be nice if the script can automatically install delve by doing 'go get <delve> && cd <delve-path> && make install' if needed or add this package by default.
This can be tackled in a separate pr when delve is commonly used by many folks.

**Comment by pravisankar** [comment]
LGTM

**Comment by marun** [comment]
@eparis Please review

**Comment by dcbw** [comment]
LGTM

File: test/extended/networking.sh [hunk 1]
```diff
@@ -20,6 +20,8 @@ source "${OS_ROOT}/hack/util.sh"
 source "${OS_ROOT}/hack/common.sh"
 os::log::install_errexit
 
+USE_DEBUGGER=${OPENSHIFT_USE_DEBUGGER:-false}
```
Comment (liggitt) [inline test/extended/networking.sh]: why have two named envvars? why not just use `OPENSHIFT_USE_DEBUGGER` directly? Also, if we're inventing envvar names, I'd prefer a more targeted one (`NETWORK_DEBUG=...` or something)

**Comment by danmcp** [comment]
@eparis Bump

File: test/extended/networking.sh (hunk 1 above)
Comment (marun) [inline test/extended/networking.sh]: Done.

**Comment by marun** [comment]
@eparis bump

**Comment by eparis** [comment]
[merge]

**Pull request merged**